decentraland/unity-explorer
# Fix: avpro fixes

## What does this PR change?
Fix 2 issues that caused exceptions for AVPRo:
1.  AutoOpen was enabled by default, leading to 2 exceptions due to opening invalid URL
2. Some URL was leading to the unavailable resources, that was fixed with prior HEAD request


## How to test the changes?
1. Goto 55,61 at GenesisCity and verify videos plays as previously
2. Additionally you can check -3,132, but video there will be some time freezing (not related to this PR)

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
